plugins/toolbox/package.json

```diff
@@ -29,6 +29,7 @@
     "postpack": "backstage-cli package postpack"
   },
   "dependencies": {
+    "@types/js-beautify": "^1.13.3",
```

Comment:
Shouldn't these be in devDependencies instead?

Reply:
yes, obviously. changed it